Sources/App/Fastlane/SlackCommand+Fastlane.swift

```diff
@@ -96,18 +98,22 @@ extension SlackCommand {
         let options = metadata.textComponents.dropFirst().joined(separator: " ")
         let branch = branch ?? metadata.value(forOption: .branch)
 
-        let parameters = ["FASTLANE": lane, "OPTIONS": options]
-        
+        let parameters: [String: CircleCIService.PipelineRequest.Parameter] = [
+            "push": .bool(false),
+            "lane": .string(lane),
+            "options": .string(options)
+        ]
+
         return try ci
-            .run(
+            .pipeline(
                 parameters: parameters,
                 project: RepoMapping.ios.repository.fullName,
                 branch: branch ?? RepoMapping.ios.repository.baseBranch,
                 on: container
             )
             .map {
                 SlackResponse("""
-                    You asked me: `\(metadata.text)`.
+                    You asked me: `\(metadata.command) \(metadata.text)`.
```

Comment:
Actually changing the command built for those aliases from `"/"` (on [line 51](https://github.com/Babylonpartners/Stevenson/pull/33/files#diff-3d1cc316946a96727edb1712aef66de2R51) and [81](https://github.com/Babylonpartners/Stevenson/pull/33/files#diff-3d1cc316946a96727edb1712aef66de2R81)) to `"/fastlane"` would make more sense, as `/hockeyapp foo bar` is just alias for `/fastlane hockeyapp target:foo bar` so that's what the `SlackCommandMetadata` we're creating to simulate such a call should look like anyway.

So making those simple changes on line 51 and 81 would fix the strangeness on this line 116 too for those cases and make everything consistent in the slack response too.

Reply:
my idea is to output the command that was called. `/fastlane hockeyapp Babylon` is not correct as it should be `/fastlane hockeyapp target:Babylon`, but `/hockeyapp Babylon` is correct.

Reply:
`metadata.text` will be `"hockeyapp target:foo"` though while the command called was `/hockeyapp foo`

Reply:
I don't really think this is important, but if you insist I change this